modules/tinymce/src/themes/silver/main/ts/ui/statusbar/Statusbar.ts

```diff
@@ -38,19 +39,53 @@ const renderStatusbar = (editor: Editor, providersBackstage: UiFactoryBackstageP
     };
   };
 
+  const renderHelpAccessibility = (): SimpleSpec => {
+    return {
+      dom: {
+        tag: 'div',
+        classes: [ 'tox-statusbar__help-text' ],
+      },
+      components: [
+        GuiFactory.text(`Press ${ConvertShortcut.convertText('Alt+0')} for help`)
+      ]
+    };
+  };
+
+  const renderRightContainer = () => {
+    const components: SimpleSpec[] = [];
+
+    if (editor.hasPlugin('wordcount')) {
+      components.push(renderWordCount(editor, providersBackstage));
+    }
+
+    if (Options.useBranding(editor)) {
+      components.push(renderBranding());
+    }
+
+    return {
+      dom: {
+        tag: 'div',
+        classes: [ 'tox-statusbar__right-container' ]
+      },
+      components
+    };
+  };
+
   const getTextComponents = (): SimpleSpec[] => {
     const components: SimpleSpec[] = [];
 
     if (Options.useElementPath(editor)) {
       components.push(ElementPath.renderElementPath(editor, { }, providersBackstage));
     }
 
-    if (editor.hasPlugin('wordcount')) {
-      components.push(renderWordCount(editor, providersBackstage));
+    const shouldRenderHelpAccessibility = editor.hasPlugin('help') && Options.useHelpAccessibility(editor);
```

Comment:
can the option be defaulted based on whether the plugin is loaded (in `Options.ts`), instead of doing it here?

Reply:
Thanks! Made changes.